tests/apollo/test_skvbc.py

```diff
@@ -32,7 +33,10 @@ def start_replica_cmd(builddir, replica_id):
     return [path,
             "-k", KEY_FILE_PREFIX,
             "-i", str(replica_id),
-            "-s", statusTimerMilli]
+            "-s", statusTimerMilli,
+            "-p" if os.environ.get('BUILD_ROCKSDB_STORAGE', "").lower() == "true" \
+                 or os.environ.get('BUILD_ROCKSDB_STORAGE', "").lower() == "on" \
```

Comment:
The following would be more elegant:
`if os.environ.get('BUILD_ROCKSDB_STORAGE', "").lower() in set(["true", "on"])`